### PR TITLE
Update "Download selected files" modal

### PR DESCRIPTION
**Pull request by adamjtaylor**
- Refines language and layout across the file download modal
- Lists the modes of file access at the top of the header
- Adds instructions and manifest download for using the gen3 client
- Lists Synapse filenames available to access
- Updates the `DataAvailabilityTable` and `FileFilterControls` use the same lock colours

### CODE REVIEW

**Comment by adamjtaylor** [comment]
@inodb @onursumer I ended up futzing with this over the weekend but think it is looking good and has both refined language and added functionality. Here are screenshots of what the download files pop-over looks like now

Screenshot 2024-11-03 at 2 23 31 PM

Screenshot 2024-11-03 at 2 23 43 PM

**Comment by inodb** [comment]
@adamjtaylor wow! Looks great! Thanks for working on getting this sorted before the jamboree! Will be very useful!

**Comment by onursumer** [comment]
Fix #632

File: packages/data-portal-explore/src/components/FileTable.tsx [hunk 1]
```diff
@@ -87,17 +90,49 @@ function generateCdsManifestFile(files: Entity[]): string | undefined {
...
+
+    // Return a single command that pulls all objects at once
+    return guids ? `gen3 --endpoint=nci-crdc.datacommons.io/ drs-pull objects ${guids}` : '';
+}
+
+function copyToClipboard(text: string) {
+    navigator.clipboard.writeText(text).then(() => {
+        alert("Copied to clipboard!");
+    });
+}
```
Comment (onursumer) [inline packages/data-portal-explore/src/components/FileTable.tsx:121]: These functions don't seem to be used anywhere. We should probably remove them.

```suggestion
```

**Review by inodb — APPROVED** [review]
Thanks so much @adamjtaylor ! Great work!

**Pull request merged**